cdm/core/src/main/java/ucar/nc2/ft2/coverage/HorizCoordSys.java

```diff
@@ -493,7 +493,7 @@ public LatLonRect calcLatLonBoundingBox() {
       maxLon = Math.max(maxLon, boundaryPoint.getLongitude());
     }
 
-    return new LatLonRect(LatLonPoint.create(minLat, minLon), LatLonPoint.create(maxLat, maxLon));
+    return new LatLonRect(LatLonPoint.create(minLat, minLon), maxLat - minLat, maxLon - minLon);
```

Comment:
The `LatLonPoint.create` normalizes the longitude to between -180 and +180 (not sure if this is good but seemed like more work to change). So previously, the constructed `LatLonRect` would have same longitude for the left and right of the rectangle. Using the other construct (with a deltaLongitude) allows the `LatLonRect` to know that the `width` of the rectangle is 360 instead of 0.